ffig/templates/_c.h.tmpl

```diff
@@ -17,6 +17,10 @@
 #define EXPORT extern "C" __attribute__((visibility("default")))
 #endif
 
+#define FFIG_DEFINE_CAPI_TYPE(module, class) struct module##_##class##_t; typedef const void* module##_##class
+FFIG_DEFINE_CAPI_TYPE({{module.name}}, {{class.name}});
+#undef FFIG_DEFINE_CAPI_TYPE
```

Comment:
I don't see the point of this macro definition and invocation. We could directly replace this with the equivalent two lines...